Automattic/jetpack
# SEO Tools: move the plugin to the plugins' directory

## Proposed changes:

Similarly to #33238, the SEO editor plugin isn't a block, so let's move it the plugins' directory for consistency.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Existing functionality should not change.

* Go to Jetpack > Settings > Traffic
* Enable the SEO feature
* Go to Posts > Add New
* Open the Jetpack plugin sidebar
* Find the SEO panel there, it should continue to work.
